File: src/main/java/org/cloudfoundry/credhub/constants/EncryptionConstants.java
package org.cloudfoundry.credhub.constants;

public class EncryptionConstants {

  public static final int NONCE_SIZE = 12;
  public static final int ENCRYPTED_BYTES = 7000;
  public static final int SALT_SIZE = 64;
  public static final int KEY_BIT_LENGTH = 256;
  public static final int ITERATIONS = 100000;
}

File: components/constants/src/main/kotlin/org/cloudfoundry/credhub/constants/EncryptionConstants.kt
package org.cloudfoundry.credhub.constants

class EncryptionConstants private constructor() {
    companion object {
        const val NONCE_SIZE = 12
        const val ENCRYPTED_BYTES = 7000
        const val SALT_SIZE = 64
        const val KEY_BIT_LENGTH = 256
        const val ITERATIONS = 100000
    }
}
